tracee/printer.go

```diff
@@ -110,8 +112,8 @@ func (p tableEventPrinter) Print(event Event) {
 	} else {
 		fmt.Fprintf(p.out, "%-14f %-16s %-6d %-20s %-16s %-6d %-6d %-6d %-16d", event.Timestamp, event.HostName, event.UserID, event.EventName, event.ProcessName, event.ProcessID, event.ThreadID, event.ParentProcessID, event.ReturnValue)
 	}
-	for _, value := range event.Args {
-		fmt.Fprintf(p.out, "%v ", value)
+	for i, value := range event.Args {
+		fmt.Fprintf(p.out, "%s: %v ", event.ArgsNames[i], value)
```

Comment:
you have only handled the table printer and not the others, but I think the other will be naturally fixed if address my other comment fro above

Reply:
Nothing to do in the others - the json printer will just marshal it as well, same for the gob which will encode as well